lib/ui/window.dart

```diff
@@ -10,6 +10,9 @@ typedef VoidCallback = void Function();
 /// Signature for [Window.onBeginFrame].
 typedef FrameCallback = void Function(Duration duration);
 
+/// Signature for [Window.onReportTimings].
+typedef TimingsCallback = void Function(List<FrameTiming> timings);
```

Comment:
this should talk about why the argument is a list and what order the list should be in